src/submission/models.py

```diff
@@ -577,13 +579,10 @@ class Article(AbstractLastModifiedModel):
         null=True,
         help_text=_('Do not use--deprecated in version 1.4.1 and later.')
     )
-    abstract = models.TextField(
+    abstract = BleachField(
         blank=True,
         null=True,
-        help_text=_('Please avoid pasting content from word processors as they can add '
-                    'unwanted styling to the abstract. You can retype the abstract '
-                    'here or copy and paste it into notepad/a plain text editor before '
-                    'pasting here.')
+        help_text=_('Copying and pasting from word processors is supported.'),
```

Comment:
Putting this comment here since I can't point at untouched lines: recently it surfaced that we need to bleach the article title field, since while not a rich-text field, we still render it as `safe`. This turns the article title into a vector for JS injection

Reply:
OK, I've changed the title field to a BleachField